imod/mf6/package.py

```diff
@@ -114,6 +119,8 @@ def _initialize_template(pkg_id):
             fname = f"gwt-{pkg_id}.j2"
         elif pkg_id in EXCHANGE_PACKAGES:
             fname = f"exg-{pkg_id}.j2"
+        elif pkg_id in API_PACKAGES:
+            fname = f"api-{pkg_id}.j2"
```

Comment:
I'm not sure we need a separate API_PACKAGES group. We are not adding more api packages are we?
Further the output file will be named api-api which is also a bit strange.
We could handle the Api package in a similair way like the ims and tdis package as it is one of a kind

Reply:
done